src/Routes/SeasonsController.ts

```diff
@@ -2,8 +2,17 @@ import * as express from "express";
 import { IDataProvider } from "../DataProviders/IDataProvider";
 import { BaseController } from "./BaseController";
 
-/*
- * Seasons controller
+/**
+ * @swagger
+ *
+ * /:
```

Comment:
route is called Seasons, so the api is /seasons

Reply:
done